src/components/tooltip/Tooltip.scss

```diff
@@ -88,6 +93,23 @@ $tooltip-offset: 4px; /* amount of space between target and tooltip arrow */
         }
     }
 
+    .tooltip-callout {
+        display: flex;
+        flex-flow: column;
+        font-weight: normal;
+        font-size: 13px;
+        line-height: 1.54;
+        letter-spacing: .3px;
```

Comment:
Is this needed or can we use the default kerning?

Reply:
yes, it's not needed, thank you